### ISSUE TITLE
Multiple attachment to cucumber report

### ISSUE THREAD

**Issue by tamil777selvan** [wswebcreation/wdio-cucumberjs-json-reporter#21]
Is there a way for attaching more than one attachment to the cucumber reports. Like error screenshot and some specific exceptions.

**Environment (please complete the following information):**
 - **Node.js version:** 10
 - **NPM version:** 6.9
 - **webdriver.io version:** 5.16.7
 - **@wdio/cucumber-framework version:** 5.16.7
 - **wdio-cucumberjs-json-reporter version:** 1.1.2

**Comment by wswebcreation** [comment]
Hi @tamil777selvan 

Have you checked the docs? This should work https://github.com/wswebcreation/wdio-cucumberjs-json-reporter#attachment

Please let me know if it works for you and if so please close this issue, if not, please provide some info about what it not working with a sample setup.

Thanks

**Comment by tamil777selvan** [comment]
@wswebcreation Thanks for the response, I have checked this https://github.com/wswebcreation/wdio-cucumberjs-json-reporter#attachment and this work fine.

Probably I would rephrase my query, Is there a way where we can add two types of attachments for a single cucumber scenario?

Ex: I need to attach two different format of log for single scenario, 1st log would be embedded in "text/html" format and the next would be "video/mp4".

**Comment by wswebcreation** [comment]
Yes, provide 2 attachments, that should work

This is what I use in my testing project

```js
After(scenarioResult => {
    // It will add the screenshot to the JSON
    cucumberJsJson.attach('After string 1');
    cucumberJsJson.attach('After string 2', 'text/plain');
    cucumberJsJson.attach(browser.takeScreenshot(), 'image/png');
});
```

Which results in 

![image](https://user-images.githubusercontent.com/11979740/69369308-24cbf800-0c9c-11ea-88e5-df14584c896b.png)


And when I expand

![image](https://user-images.githubusercontent.com/11979740/69369361-3d3c1280-0c9c-11ea-8a9e-579668836380.png)

Can you please verify?

**Comment by tamil777selvan** [comment]
@wswebcreation Thanks for the update. This works as expected. But also seems to get flaky. Will close the issue.
